test/test-growpart

```diff
@@ -18,13 +18,22 @@ clearparts() {
 	# read /proc/partitions, clearing any partitions on dev (/dev/loopX)
 	local dev="$1"
 	local short=${dev##*/} parts="" part=""
+	local removed=0
 	parts=$(awk '$4 ~ m { sub(m,"",$4); print $4 }' \
 		"m=${short}p" /proc/partitions)
 	[ -z "$parts" ] && return
 	echo "clearing parts [$parts] from $dev"
 	for part in $parts; do
 		echo "delpart $LODEV $part"
-		delpart $LODEV $part
+		for retry in $(seq 1 5); do
+			delpart $LODEV $part && removed=1 && break
+			echo "Failed, retrying - $retry"
+			sleep 2s
```

Comment:
What's the delpart error?  Instead of sleeping, should delpart be waiting until the device has removed?  Should it use dmsetup to remove the connection?

Reply:
> What's the delpart error? Instead of sleeping, should delpart be waiting until the device has removed? Should it use dmsetup to remove the connection?

see #24 which did this hunk differently.

Reply:
Indeed, #24 evolved out of a discussion by me and @paride - so I can drop that commit from the series and keep the rest.

Reply:
Rebased, the one change we solved differently dropped and pushed to this branch.
Ready for re-review and hopefully acceptance :-)